recipes/geehydro/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "geehydro" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4ab95381b33bdb06cdf8a426c6e9a5117c26307e1ee7ec10b1ac018791db20e6
+
+build:
+  number: 0
+  entry_points:
+    - geehydro=geehydro.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click
+    - earthengine-api
+    - folium
+    - pip
+    - python
+  run:
+    - click
+    - earthengine-api
+    - folium
+    - python
+
+test:
+  imports:
+    - geehydro
+  commands:
+    - geehydro --help
+
+about:
+  home: "https://github.com/giswqs/geehydro"
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
Please add license file. If it is not part of the tarball, you can copy it from the repo.